cypress/e2e/signUp.cy.js

```diff
@@ -1,12 +1,71 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import SignUpPage from '../support/pages/signUp.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import faker from 'faker';
 
 describe('Sign Up page', () => {
+  let name;
+  let email;
+  let password;
+  const signUpPage = new SignUpPage();
+
   before(() => {
+    cy.task('db:clear');
+  });
+
+  beforeEach(() => {
+    cy.visit('/#/register');
+    name = faker.name.firstName();
+    email = faker.internet.email();
+    password = faker.internet.password();
+  });
+
+  it('should create a new user', () => {
+    signUpPage.fillName(name);
```

Comment:
It's better to use a single method for filling the form. It increases code readability and reusability. Consider changing these methods into a single one, like `signUpPage.fillForm(name, email, password)`.

Reply:
done